docs/architecture/deep-dives/state/index.md

```diff
@@ -172,6 +200,13 @@ class FolderService {
 
 ### `ActiveUserState<T>`
 
+:::warning
+
+`ActiveUserState` has problems with consider not using it anymore,
```

Comment:
⛏️ Looks like you a word. "`ActiveUserState` has race condition problems. Do not use it for updates and consider transitioning your code to SingleUserState instead."